### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'start'

### ISSUE THREAD

**Issue by FlimsyFox** [hydrusnetwork/hydrus#175]
``` AttributeError
'NoneType' object has no attribute 'start'
Traceback (most recent call last):
  File "include\HydrusThreading.py", line 274, in run
    callable( *args, **kwargs )
  File "include\ClientGUIDialogs.py", line 1752, in THREADParseImportablePaths
    mime = HydrusFileHandling.GetMime( path )
  File "include\HydrusFileHandling.py", line 212, in GetMime
    if HydrusVideoHandling.HasVideoStream( path ):
  File "include\HydrusVideoHandling.py", line 93, in HasVideoStream
    info = Hydrusffmpeg_parse_infos( path )
  File "include\HydrusVideoHandling.py", line 194, in Hydrusffmpeg_parse_infos
    result['video_fps'] = float(line[match.start():match.end()].split(' ')[1])
AttributeError: 'NoneType' object has no attribute 'start'
```

Happens during an import of a large folder. Stops the import process in its tracks.

**Comment by ScoreUnder** [comment]
As a temporary workaround until the Hydrus dev gets to this, try moving any `wma`/`wmv`/`asf` files out of the directory before starting the import.

**Comment by mserajnik** [comment]
Hey, issues here have been reenabled and will be used for bug reports/feature requests in the future.

We are currently clearing out all the existing issues from 2017 and prior.
If your bug report or feature request is still relevant, please create a new issue (or use one of the other channels; see https://hydrusnetwork.github.io/hydrus/).
